testing-proto/src/main/proto/io/grpc/testing/integration/test.proto

```diff
@@ -47,6 +47,11 @@ service TestService {
   // One request followed by one response.
   rpc UnaryCall(SimpleRequest) returns (SimpleResponse);
 
+  // One request followed by one response. Response has cache control
```

Comment:
This behavior is set to change soon, and the interop test protos have already diverged enough between here and grpc/grpc. Will update this comment once I add these files into grpc-proto and have a standardized location for the 
"ground truth".